### PR TITLE
Refactor css

### PR DESCRIPTION
**Pull request by aliroemhildt**
#### What's this PR do?
- refactor CSS
- add page title element

#### Where should the reviewer start?
- review css file (we reordered)
- look at nav section of html

#### How should this be manually tested?
open site

#### Any background context you want to provide?
n/a

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
n/a

#### Questions:
n/a

### CODE REVIEW

**Review by cagallo — APPROVED** [review]
I love this, great for UI!

**Review by kammon10 — APPROVED** [review]

File: src/index.html [hunk 1]
```diff
@@ -14,6 +14,9 @@ <h1>what's cookin'?</h1>
       <input class="input" type="text" id ="searchInput" placeholder="Search recipes...">
       <button id ="searchRecipes">search</button>
     </div>
+    <div class="page-title-wrapper">
```
Comment (anewb87) [inline src/index.html:17]: YES! So much easier to know where a user is.

File: src/styles.css [hunk 2]
```diff
@@ -1,4 +1,3 @@
-/* font-family: 'Lora', serif; */
 /* ~~~~~~~~~~ BASIC ELEMENTS ~~~~~~~~~~~~ */
```
Comment (anewb87) [inline src/styles.css:1]: Really appreciate the headers to each section delineating what's going on in there.

**Review by anewb87 — APPROVED** [review]
Final refactor on CSS. Lookin and cookin good!

**Pull request merged**